src/main/java/org/wso2/carbon/module/swiftiso20022/validation/common/ValidationResult.java

```diff
@@ -44,6 +44,10 @@ public boolean isValid() {
         return this.isValid;
     }
 
+    public boolean isNotValid() {
+        return !this.isValid;
```

Comment:
Shall we add method comments too?

```suggestion
        return !this.isValid();
```

Reply:
Changed the method and add method comment in commit 4e687a247135e6bc9d43274b4f9f97756cf64480